{"additions": 1, "auther_ref": "fc-pythia8-20190912", "auther_sha": "11acf1aee7bc34a45d630f514bb60c60903551ec", "author": "fabiocos", "body": "Addition of cms-externals/pythia8#14 by S. Mrenna fixing the issue cms-sw/cmssw#27918", "branch": "IB/CMSSW_11_0_X/master", "changed_files": 1, "comments": 1, "commits": 1, "created_at": "1568271965", "deletions": 1, "labels": ["comparison-pending", "externals-pending", "orp-pending", "pending-signatures", "tests-pending"], "number": 5229, "release-notes": [], "review_comments": 0, "state": "open", "title": "Pythia8.240 fix for re-initialization", "updated_at": "1568271991", "user": "fabiocos"}